### PR TITLE
feat: UPE preview onboarding first step

### PR DESCRIPTION
**Pull request by frosso**
Helps with https://github.com/Automattic/woocommerce-payments/issues/2170

#### Changes proposed in this Pull Request

I decided to split a bigger PR. Other steps of the onboarding flow will be done separately.
This just makes some UI changes on the first onboarding step.
No functionality has been changed, just text and CSS has been updated - no tests have been added.

New visual:
Screen Shot 2021-07-14 at 10 54 34 AM

#### Testing instructions

- Ensure you have `_wcpay_feature_account_overview_task_list` with `'1'` in your `wp_options`
- Ensure "Enable UPE settings changes" is enabled
- Ensure "Enable UPE checkout" is disabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- UI is updated with 2 boxes

-------------------

- [ ] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

### CODE REVIEW

File: client/additional-methods-setup/upe-preview-methods-selector/enable-upe-preview-task.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
+.enable-upe-preview {
+	&__body {
+		&.is-active {
+			// there's so much text that on mobile the text overflows the max-height set on the collapsible body.
```
Comment (frosso) [inline client/additional-methods-setup/upe-preview-methods-selector/enable-upe-preview-task.scss:4]: Without this on mobile:
Screen Shot 2021-07-14 at 10 47 30 AM

File: client/additional-methods-setup/wizard/collapsible-body.js [hunk 2]
```diff
@@ -10,17 +10,16 @@ import classNames from 'classnames';
 import WizardTaskContext from './task/context';
 import './collapsible-body.scss';
 
-const CollapsibleBody = ( { children } ) => {
+const CollapsibleBody = ( { className, ...restProps } ) => {
```
Comment (frosso) [inline client/additional-methods-setup/wizard/collapsible-body.js:13]: Added a `className` prop to allow modifications

File: client/additional-methods-setup/upe-preview-methods-selector/enable-upe-preview-task.js [hunk 3]
```diff
@@ -20,26 +24,103 @@ const EnableUpePreviewTask = () => {
...
-					) }
+					{ interpolateComponents( {
+						mixedString: __(
+							'Get early access to additional payment methods and an improved checkout experience, ' +
+								'coming soon to WooCommerce payments. {{learnMoreLink /}}',
+							'woocommerce-payments'
+						),
+						components: {
+							learnMoreLink: (
+								<a href="?TODO">
```
Comment (frosso) [inline client/additional-methods-setup/upe-preview-methods-selector/enable-upe-preview-task.js:58]: We don't have this yet. Here's the plan if you're curious: https://github.com/Automattic/woocommerce-payments/issues/2170#issuecomment-880008439

File: client/additional-methods-setup/upe-preview-methods-selector/index.scss [hunk 4]
```diff
@@ -1,4 +1,4 @@
 .upe-preview-methods-selector {
-	max-width: 680px;
+	max-width: 700px;
```
Comment (frosso) [inline client/additional-methods-setup/upe-preview-methods-selector/index.scss:2]: Matches the designs a bit more closely in this flow

**Comment by harriswong** [comment]
👀

**Review by harriswong — APPROVED** [review]
Tested and confirmed `wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods` matches design. 
LGTM

**Pull request merged**